internal/template/template.go

```diff
@@ -44,6 +45,34 @@ func GenerateVarsTf(opts VarsOpts, path string) error {
 	return nil
 }
 
+func GenerateConfigFile(opts ConfigOpts) error {
+	configPath, _ := os.Getwd()
```

Comment:
Should this take config path into the account (and not just use current directory)? Precedence may be the follwing:
1. Check if config variable is set? (let's say we use something like `ize init --config=.infra/env/dev/ize.hcl` (or `IZE_CONFIG_DIR` variable)
2. Use current directory

Honestly, it seems like we should determine whether it's the value that is passed via config or cwd in the actual global config, and just reference it here (same way as we would when we read it). 

Thoughts?